shared/js/background/classes/score.es6.js

```diff
@@ -1,10 +1,11 @@
+const tldjs = require('tldjs')
 const tosdr = require('../../../data/tosdr')
 const constants = require('../../../data/constants')
-const tosdrRegexList = Object.keys(tosdr).map(x => new RegExp(x))
+const utils = require('../utils.es6')
+const tosdrRegexList = Object.keys(tosdr).map(x => new RegExp(`(^)${tldjs.getDomain(x)}`)) // only match domains, and from the start of the URL
```

Comment:
What is getDomain used for and what about matching subdomains?

Reply:
@jdorweiler It's used to match the domain. If there are cases where we need to match the subdomain (are there?) I can pair it with getSubDomain() like [here](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/922a698178bbc40447904a6b8331bba8e91afe4b/shared/js/ui/models/whitelist.es6.js#L43=)